### ISSUE TITLE
Can not import MedtronicMachine

### ISSUE THREAD

**Issue by mortlind** [pazaan/decoding-contour-next-link#18]
I am on commit efb625227ebbda41cb7956e4fb1421cbfecf4afb, the HEAD at the time of writing. I get the following error when trying to get the hmac:

```
python get_hmac_and_key.py my_carelink_username
Traceback (most recent call last):
  File "get_hmac_and_key.py", line 16, in <module>
    from read_minimed_next24 import Config, MedtronicMachine
ImportError: cannot import name MedtronicMachine
```

And right it is, there is no name `MedtronicMachine` defined in `read_minimed_next24.py`

**Comment by paulokow** [comment]
it's fixed on my fork https://github.com/paulokow/decoding-contour-next-link
PR #17  is still open

**Comment by mortlind** [comment]
Hi paulolkow, I tried you fork, but it gives me different issues. I will submit an issue in your fork. For this issue, in the original repo, I traced that `response.raw.read()` on line 63 of `get_hmac_and_key.py` returns '' which causes `self.decodeResponse( ... )` on line 69 to throw a marshalling exception.

**Comment by paulokow** [comment]
@mortlind, looks like problems with reading data from decocare. It's a part of original code from @pazaan and I did not dig too much into that part as it worked form out of the box after fixing the  import  of MedtronicMachine

**Comment by paulokow** [comment]
Fixed with e8c59fb8831ebfdbec37b071a92c352b421d112c
